### PR TITLE
Report the last cgroup destroy event.

### PR DESCRIPTION
**Pull request by vmarmol**
/cc @rjnagal

### CODE REVIEW

File: container/raw/inotify_watcher.go [hunk 1]
```diff
@@ -55,46 +51,51 @@ func (iw *InotifyWatcher) AddWatch(containerName, dir string) (bool, error) {
...
 }
 
-// Remove watch from the specified directory. Returns if the container was already being watched.
+// Remove watch from the specified directory. Returns if this was the last watch on the specified container.
 func (iw *InotifyWatcher) RemoveWatch(containerName, dir string) (bool, error) {
 	iw.lock.Lock()
 	defer iw.lock.Unlock()
 
-	alreadyWatched := iw.containersWatched[containerName]
+	cgroupsWatched := iw.containersWatched[containerName]
```
Comment (rjnagal) [inline container/raw/inotify_watcher.go]: We need to check if cgroupsWatched returns nil.

**Comment by rjnagal** [comment]
I think we can redo inotifyWatcher logic to provide a smaller interface later. I am fine with the current code in the meantime.

File: container/raw/inotify_watcher.go (hunk 1 above)
Comment (vmarmol) [inline container/raw/inotify_watcher.go]: If it returns nil, all the operations we use here are handled (get, delete, and len). Went ahead and made it explicit though to make it clearer.

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**